lib/report_generator.rb

```diff
@@ -23,18 +27,17 @@ def generate(query_id, query_params, recipients)
     end
 
     def filter_recipients_by_query_access(recipients, query)
```

Comment:
Minor but I think this method should probably be private now that it doesn't do the basic guardian checks.